src/services/production.ts

```diff
@@ -85,6 +86,15 @@ export const services: ServiceDefinition[] = [
     ],
   }],
 
+  // Router
+  [$.router, {
+    service: createRouter,
+    arguments: [
+      $.routes,
+      $.store,
+    ],
+  }],
+
```

Comment:
This is pulled into our test `main` file, not yet our application `main` file, but we may as well do the same in our application `main` file soon.